Source/Scene/ModelExperimental/GeometryPipelineStage.js

```diff
@@ -31,7 +31,7 @@ var GeometryPipelineStage = {};
  * @private
  */
 GeometryPipelineStage.process = function (renderResources, primitive) {
-  var attributeIndex = 0;
+  var attributeIndex = renderResources.attributeIndex;
```

Comment:
(below the fold): I see `attributeIndex = attributeIndex + 1`. `+=` would be a little more concise.